src/Elastic.Apm/Sampler.cs

```diff
@@ -20,7 +20,8 @@ namespace Elastic.Apm
 	{
 		private readonly long _higherBound;
 		private readonly long _lowerBound;
-		private readonly double _rate;
+
+		public double Rate { get; }
```

Comment:
With exposing the rate as a property, it'd probably be useful to update the Sampler tests that assert rounding to use this rather than the `.ToString()` representation. Doesn't need to be part of this PR though

Reply:
Updated the `SamplerTests.Rate_Precision_Should_Be_Rounded_To_Four_Decimal_Places` to also assert on the `Rate` property. Wasn't sure if we completely want to remove the `ToString()` overload (I think it's useful), so I also kept asserting on `ToString()`.